com.unity.ml-agents/Tests/Editor/PublicAPI/Unity.ML-Agents.Editor.Tests.PublicAPI.asmdef

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Unity.ML-Agents.Editor.Tests.PublicAPI",
+    "references": [
+        "Unity.ML-Agents.Editor",
+        "Unity.ML-Agents",
+        "Barracuda",
+        "Unity.ML-Agents.CommunicatorObjects"
+    ],
+    "optionalUnityReferences": [
+        "TestAssemblies"
+    ],
+    "includePlatforms": [
+        "Editor"
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": false,
+    "overrideReferences": true,
+    "precompiledReferences": [
+        "System.IO.Abstractions.dll",
+        "System.IO.Abstractions.TestingHelpers.dll",
+        "Google.Protobuf.dll"
```

Comment:
Not sure we will need protobuf for testing the public API

Reply:
Like I said before, I just copied this from the editor tests. I can try removing this from here, but it's also not something that needs to be optimized.

Reply:
Updated.